templates/actor/coterie-sheet.html

```diff
@@ -0,0 +1,38 @@
+<form class="{{cssClass}} flexcol {{#if locked}}locked{{/if}} coterie-sheet" autocomplete="off">
```

Comment:
Coteries, Ghoul and Mortal sheets need a better design.

This is functionnal but not very stylish.

Reply:
![coterie-sheet](https://user-images.githubusercontent.com/1277339/123556472-b9a5aa00-d78b-11eb-879b-19dc3e0ee5f4.jpeg)